_posts/2018/01/2018-01-17-sponsor-carpentrycon.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+authors: ["Belinda Weaver"]
+title: "An Open Letter to Companies - Sponsor CarpentryCon 2018"
+date: 2018-01-10
+time: "00:00:00"
+category: ["CarpentryCon", "Sponsorship", "Community Building"]
+---
+
+The inaugural [CarpentryCon 2018](http://www.carpentrycon.org/) will take place on 30 May - 1 June at [University College Dublin](http://www.ucd.ie/).
+
+CarpentryCon 2018 aims to skill up the next generation of research leaders. 
+
+Unlike conferences that focus on advances within a discipline, CarpentryCon will teach the practical skills people need to lead 21st century research within academia and industry.
+
+### Why should your organisation sponsor this event?
+
+- Expose your products and services to a motivated audience eager to do cutting edge research and robust data science.
+- Support the development of the diverse and skilled workforce industry needs to innovate
+- Gain useful insights from researchers about the problems they are trying to solve
+- The Carpentries have great 'brand' recognition worldwide. Reach our people
+- Help foster best practice around open, reproducible science - this benefits everyone
+- Support skilling up of researchers from the global South - they will take your message home
+
+Do not miss out on sponsoring this **landmark event**. 
+[Express interest or start a conversation here](https://docs.google.com/forms/d/e/1FAIpQLSedlt68CXVmyVJ4DEI8P9nfAXhGYbTHA9YgFQYomXjzzZDJOg/viewform).
+
+### What is CarpentryCon all about?
+
+CarpentryCon 2018 aims to skill up the next generation of research leaders.
+
+This might mean learning advanced R or Python, skilling up on High Performance Computing, or figuring out how to lead a lab or 
+a big research project. Sessions will be hands-on, and attendees will leave with practical skills they can immediately use 
+in their research or careers.
+
+The Carpentries teach introductory material - enough to get people started with coding and data science tools. 
+But to innovate, researchers need more. Our  people want  to hear about tools or services that can take their 
+research to the next level. 
+
+*So there is real synergy there between your organisation's goals and ours.*
+
+CarpentryCon offers a great opportunity to put what you offer in front of a highly motivated and receptive global audience. 
+Attendees will be a mixture of early career researchers and more senior academics, so the potential is there for great conversations 
+with people about how they can best leverage the services you offer to solve complex research problems. 
+
+Want to help foster research workforce diversity? We plan to provide travel scholarships for people to attend from the Global South. 
```

Comment:
Scholarships will be for more than that region. Could it be instead "Want to help foster workforce diversity and bring new communities into computing? We plan to provide travel scholarships for people from many geographies and communities to participate, including the global South."